org.ekstep.question-1.0/editor/style.css

```diff
@@ -266,7 +266,6 @@ color: white;
 }
 
 .formDiv{
-    overflow: auto;
```

Comment:
This is not the right solution to remove scrollbar. Why scroll-bar is showing for question-form. Verify that & fix that div height or inner elements.

Reply:
I verified, the inner scroll is appearing from each selected template, and the outer scroll has appeared from question plugin. Both parent and child div using a scroll bar. 
The question plugin parent div has a fixed height, hence I removed outer scroll, and tested it's not causing any problem.
@vinukumar-vs